pyramid/httpexceptions.py

```diff
@@ -121,10 +122,11 @@
 
 The subclasses of :class:`~_HTTPMove`
 (:class:`~HTTPMultipleChoices`, :class:`~HTTPMovedPermanently`,
-:class:`~HTTPFound`, :class:`~HTTPSeeOther`, :class:`~HTTPUseProxy` and
-:class:`~HTTPTemporaryRedirect`) are redirections that require a ``Location``
-field. Reflecting this, these subclasses have one additional keyword argument:
-``location``, which indicates the location to which to redirect.
+:class:`~HTTPFound`, :class:`~HTTPSeeOther`, :class:`~HTTPUseProxy`
+:class:`~HTTPTemporaryRedirect` and :class: `~HTTPPermanentRedirect) are
```

Comment:
A comma was dropped, and while we're at it, let's use the Oxford comma.

```rst
:class:`~HTTPFound`, :class:`~HTTPSeeOther`, :class:`~HTTPUseProxy`,
:class:`~HTTPTemporaryRedirect`, and :class: `~HTTPPermanentRedirect) are
```

Reply:
Nice catch, thanks.

Updated to fix the dropped comma and also added the oxford comma.